neps/nep-0488.md

```diff
@@ -301,281 +301,24 @@ All parameters were taken from[^15],[^51] and [^14], all of them consistent betw
-
-- $A$, $B$ -- the parameters of Weierstress curve
-- $Z$ the element of the field $F$ (in our case $F_p$ or $F_{p^2}$), which meeting the below criteria:
-  - Z is non-square in F
-  - $Z \ne -1$ in F
-  - The polynomial $g(x) - Z$ is irreducible over F
-  - $g(\frac{B}{Z \cdot A})$ is square in $F$.
-
-For points $u$ and $-u$ we will get the same $x$-coordinate. We will define the $y$ sign the same as $u$ sign(see the definition of sign in sections "Compressed points on curve E(Fp)" and "Compressed points on curve E'(Fp2))".
-
-The case when $Z^2 \cdot u^4 + Z \cdot u^2 = 0$ should be handled separately. In that case set $x_1 = \frac{B}{X \cdot A}$,
-which guarantees that $g(x_1)$ is square by the condition on Z given above.
-
-The algorithm:
-
-```text
-fn map_to_curve_simple_swu(u) {
-    let tv1 = 1 / (Z^2 * u^4 + Z * u^2);
-    let x1 = (-B / A) * (1 + tv1);
-    if tv1 == 0 {
-        x1 = B/(Z*A);
-    }
-    
-    let gx1 = x1^3 + A*x1 + B;
-    let x2 = Z * u^2 + x1;
-    let gx2 = x2^3 + A*x2 + B;
-    let (x, y) = if is_squere(gx1) {
-        (x1, sqrt(gx1));
-    } else {
-        (x2, sqrt(gx2));
-    }
-    
-    if sign(u) != sign(y) {
-       y = -y;
-    }
-    
-    return (x, y);
-}
-
-```
-
-***Step 2: Map to target curve***
-
-The function `iso_map(x_inner, y_inner)` map the points from isogenous curve E_inner with no-zero coefficients to our
-target elliptic curve E. 
-
-Wahby and Boneh[^57] show how to adopt the algorithm for case when elliptic curve have zero A or B. The idea: map
-the F element to isogenous elliptic curve E_inner by the method from previous section, and then transform E_inner curve
-to our target curve E by knowing isogeny. 
-
-E_inner curve with isogeny will be defined in the "Parameters" section.
-
-Note that `iso_map` is a group homomorphism. This mean `iso_map(p) + iso_map(q) = iso_map(p + q)`. So, the correspondent optimization can be applied.
-
-If any denominator in rational functions used in `iso_map` function equal to zero, the `iso_map` function
-must return identity point on E.
-
-***Step 3: Map to subgroup(cofactor cleaning)***
-
-In this section we will describe the function `clear_cofactor((x, y))`. This function transfer a point from $E(F_p)$ or $E'(F_{p^2})$ 
-into a point in $G_1 \subset E(F_p)$ or $G_2 \subset E'(F_{p^2})$ respectively. 
-
-To do that, we should multiply the point on the curve by effective cofactor `h_eff`. Multiplication on `h_eff` gives the
-results in the same subset as multiplication on cofactor, but can be executed must faster. The value of `h_eff` will be given in
-the following sections.
-
-```text
-fn clear_cofactor(P) {
-    h_eff * P
-}
-
-```
-
-***Full algorithm***
-
-In this section, the pseudocode for the full mapping algorithm is shown. All the functions are described in the sections above.
+This functions maps the field element($F_p$ or $F_{p^2}$) to the subgroup of elliptic curve($G_1 \subset E(F_p)$ or $G_2 \subset E'(F_{p^2})$).
```

Comment:
Suggestion: These functions map a field element in $F_p$ or $F_{p^2}$ to the corresponding subgroup $G_1 \subset E(F_p)$ or $G_2 \subset E'(F_{p^2})$.

Reply:
fixed: https://github.com/near/NEPs/pull/488/commits/60a4a8099e1c89c25343bcd656fba575ad51a422